x-pack/plugins/cases/public/components/markdown_editor/plugins/lens/parser.ts

```diff
@@ -1,77 +0,0 @@
-/*
```

Comment:
This was duplicated in the cases/common directory so we'll just use that one instead. Unless it's not allowed to reference the common directory from within the public directory 🤔  it seems to be working though.